tests/integration2/pilot/sidecar_api_test.go

```diff
@@ -53,7 +50,7 @@ func TestSidecarListeners(t *testing.T) {
 	}
 
 	// Test the empty case where no config is loaded
-	err = pilotInst.WatchDiscovery(time.Second*10,
+	err = pilotInst.WatchDiscovery(time.Second*30,
```

Comment:
That's really high? Were you seeing issues or was this just to enable running in a debugger

Reply:
I ran into quite a few timeout failures with 10 and a few with 20. 30 seems to be stable.

We should rationalize and standardize some of these timeouts. Internally, the test framework uses the "CIMode" flag to utilize different behavior depending on whether the framework is running locally, or in a CI system. One of the reasons for that was to adjust timeouts. Timeouts that are typically used in CI systems makes local debugging pretty painful. 

We can make this more granular, and come up with a dilation factor, as well. It may help with differences seen between various machines.